code/game/machinery/kitchen/processor.dm

```diff
@@ -244,11 +244,11 @@
 	src.visible_message("<span class='notice'>[src] is done.</span>", \
 		"You hear [src] stop")
 
-/obj/machinery/processor/attack_ghost(mob/user as mob)		
-	user.examination(src)		
-		
+/obj/machinery/processor/attack_ghost(mob/user as mob)
+	user.examination(src)
+
 /obj/machinery/processor/MouseDrop_T(atom/movable/O as mob|obj, mob/user as mob)
-	if(!ishuman(user) && !isrobot(user)) //No ghosts or mice
-		return 
-		
+	if(!user.has_hand_check() && !isrobot(user)) //No ghosts or robits
```

Comment:
if you DON'T want robots to do it, change this to a ||, but I see no problem with borgs/mommis doing it.